### ISSUE TITLE
Clicking on the border generates ColorControl.Program error (v971)

### ISSUE THREAD

**Issue by Amoeba00** [Maassoft/ColorControl#287]
Just upgraded to v971 from 970
Also, noticed the scaling is a bit off and the words are no longer auto-adjusting (with/without GDI scaling enabled)

Discovered that when I click on the border - an error is generated.  Basically, if you click on any part that's not the tab - the error is generated.

![image](https://github.com/Maassoft/ColorControl/assets/39176113/5b6eb7ff-707f-40d3-a7d2-8e83e2017c38)

Everything else in log is green:

![image](https://github.com/Maassoft/ColorControl/assets/39176113/122d0c1f-e06b-4f38-9df6-4d738c3f1c54)

Text:
```
Exception in thread: Object reference not set to an instance of an object.
   at ColorControl.MainForm.MainForm_Click(Object sender, EventArgs e) in .\MainForm.cs:line 1294
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at InvokeStub_SendOrPostCallback.Invoke(Object, Object, IntPtr*)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
   at ColorControl.Program.GlobalThreadExceptionHandler(Object sender, ThreadExceptionEventArgs e) in .\Program.cs:line 321
   at ColorControl.Program.MainAsync(String[] args) in .\Program.cs:line 162
   at ColorControl.Program.Main(String[] args) in .\Program.cs:line 51
```
Screenshot:
![image](https://github.com/Maassoft/ColorControl/assets/39176113/917e3545-c950-4751-8693-182fcdd4ca23)

**Comment by Maassoft** [comment]
The error is caused by test code for the Nvidia controller that I forgot to remove. It will crash if you don't have a Nvidia GPU.
I'm not sure what happened with the scaling, nothing is changed between 9.7.0.0 and 9.7.1.0 that could interfere with the scaling of the UI. Maybe caused by a Windows update or something else? I don't use scaling myself so those issues are easily missed then. I will increase the widths of these group boxes, there's enough room anyway.

**Comment by Amoeba00** [comment]
Ahh - Funny - when I started using your app I was on Team Green (2070S), but then upgraded to Team Red (7900xtx).  Also, I only noticed it because of those errors - it may have been there for a while and I just never noticed it on recent versions or maybe it only happens on the Radeon cards.

**Comment by Amoeba00** [comment]
Confirmed fixed with v9.7.2 - thanks!
